docs/version0.44.md

```diff
@@ -76,6 +77,12 @@ The `jcmd` tool also now supports specifying `0` as a VMID to target all VMs.
 
 For more information, see [Java diagnostic command (`jcmd`) tool](tool_jcmd.md).
 
+### Change in the Direct Dump Reader plug-ins code
+
+The Direct Dump Reader (DDR) code enables the VM to read the system dump data by using the OpenJ9 Diagnostic Tool Framework for Java (DTFJ) API or the [`jdmpview`](tool_jdmpview.md) tool. The DDR code provides many built-in commands. You can extend the DDR code, adding your own commands, by writing plug-ins. These user plug-ins might contain code that has `J9BuildFlags` reference to read build flags in the system dump data. These flags are related to the configuration of the VM that produced a specific system dump. When the `J9BuildFlags` is generated for each build, the raw name of the build flags might be used in the plug-in code rather than the `cname` that appears in `j9cfg.h` (derived from `j9cfg.h.ft1`). Therefore, mapping of flag names in the plug-in code with the names that are specified in `j9cfg.h` was inconsistent. Introducing new flags meant either extra complexity or that some flag tests might not apply to all system dumps (because the flag name in `J9BuildFlags` was different at different times).
+
+Now, the flag names that are used in the DDR code to refer to the fields of `J9BuildFlags` are changed. The flag names are now those names that appear in C or C++ code, independent of the VM that produced a system dump. To reduce complexity and improve the uniformity of how the DDR code uses the flags, you must use the names that are specified in `j9cfg.h` as flag names in the plug-in code. For example, `env_data64` is now referred to as `J9VM_ENV_DATA64`.
```

Comment:
I suggest this paragraph go first, followed by the lengthy description. The description is only needed for those who don't already understand, or those who want more details.

Reply:
Usually, the strategy that we use is to explain the problem first and then explain the solution or the change that was made to mitigate that problem. Maybe I can try and reduce the first paragraph?

Reply:
Changing the title may be good enough.

Reply:
> Usually, the strategy that we use is to explain the problem first and then explain the solution or the change that was made to mitigate that problem.

Why do we do it this way? Perhaps it should be reversed. It seems more important to describe the change first with the explanation afterwards, since the explanation is optional.

Reply:
It is propounded in technical writing that users prefer to know what the problem was and then read about the solution for it. If we had a separate topic to give this explanation, then I could have given a generic explanation in this What's new topic. And like you had suggested, I have changed the title to be more specific about the change, so I guess that should address your concern? Thanks!